package.json

```diff
@@ -27,6 +27,7 @@
     "@babel/runtime": "^7.7.7",
     "@testing-library/react": "^9.1.3",
     "@types/jest": "^25.1.2",
+    "@types/mdx-js__react": "^1.5.1",
```

Comment:
@hasparus https://github.com/system-ui/theme-ui/pull/917/files#r425106843 

I've added it here since it's a devDep and we're in a monorepo environment

Reply:
We're building a package though.

Our emitted types contain types imported from _@types/mdx-js__react_. (`MDXProviderComponents`)

We either install it for the user, or we require installing it in docs and peerDeps.
If we just leave it here, sidenav props `components` will be either `any` or fail the build for the user depending on their `skipLibCheck` setting.

Reply:
yeah you're right 👍 I'll update it

Reply:
Updated